cmd/ocp.go

```diff
@@ -80,9 +85,9 @@ func openShiftCmd() *cobra.Command {
 		if checkHealth && (cmd.Name() != "cluster-health" || cmd.Name() == "index") {
 			ocp.ClusterHealthCheck()
 		}
-		workloadConfig.ConfigDir = configDir
 		kubeClientProvider := config.NewKubeClientProvider("", "")
-		wh = workloads.NewWorkloadHelper(workloadConfig, &ocpConfig, kubeClientProvider)
+		workloadDir := filepath.Join(rootDir, cmd.Name())
+		wh = workloads.NewWorkloadHelper(workloadConfig, &ocpConfig, workloadDir, metricsProfilesDir, alertsDir, workloadDir, kubeClientProvider)
```

Comment:
Why are we reusing the `workloadDir` for the scripts dir?

Reply:
Currently, the scripts are:
```
cmd/config/virt-migration/check.sh
cmd/config/virt-clone/check.sh
cmd/config/virt-ephemeral-restart/check.sh
cmd/config/virt-capacity-benchmark/check.sh
```
But, they are all the same. `diff` returns nothing. So, they can be merged into one in a new `scripts` directory.